### ISSUE TITLE
Issue with coding Season 14 contestant outcomes

### ISSUE THREAD

**Issue by adityac95** [svmiller/dragracer#6]
Thanks for publishing this dataset, it's super fun to play with! However, there's an issue with Season 14: I'm not seeing any high or low placements; they're all showing up as SAFE. Can this be fixed?

**Comment by svmiller** [comment]
yeah, it can. Probably a lazy scrape job by me. I'll look into it and see what happened.
